### PR TITLE
dmctl: update the ouput of `operate-worker`

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What problem does this PR solve? 

if `result` of `operate-worker` is true, will print nothing

### What is changed and how it works?

fix it

### Check List 

Tests 

 - Integration test

### CODE REVIEW

File: dm/ctl/master/operate_mysql_worker.go [hunk 1]
```diff
@@ -72,5 +72,7 @@ func operateMysqlWorkerFunc(cmd *cobra.Command, _ []string) {
 
 	if !resp.Result {
 		common.PrintLines("operate worker failed:\n%v", resp.Msg)
+	} else {
+		common.PrettyPrintResponse(resp)
```
Comment (csuzhangxc) [inline dm/ctl/master/operate_mysql_worker.go]: even `resp.Result`, can we `common.PrettyPrintResponse(resp)` here?

**Comment by kennytm** [comment]
Test failure caused by data race

```
[2020-01-15T02:49:37.556Z] WARNING: DATA RACE
[2020-01-15T02:49:37.556Z] Write at 0x00c0004ee000 by goroutine 72:
[2020-01-15T02:49:37.556Z]   github.com/pingcap/dm/dm/worker.(*Server).Start.func1()
[2020-01-15T02:49:37.556Z]       /home/jenkins/agent/workspace/dm_ghpr_test/go/src/github.com/pingcap/dm/dm/worker/server.go:102 +0xff
[2020-01-15T02:49:37.556Z] 
[2020-01-15T02:49:37.556Z] Previous write at 0x00c0004ee000 by goroutine 56:
[2020-01-15T02:49:37.556Z]   github.com/pingcap/dm/dm/worker.(*Server).Start()
[2020-01-15T02:49:37.556Z]       /home/jenkins/agent/workspace/dm_ghpr_test/go/src/github.com/pingcap/dm/dm/worker/server.go:154 +0xd24
[2020-01-15T02:49:37.556Z]   github.com/pingcap/dm/cmd/dm-worker.main()
[2020-01-15T02:49:37.556Z]       /home/jenkins/agent/workspace/dm_ghpr_test/go/src/github.com/pingcap/dm/cmd/dm-worker/main.go:80 +0x6c2
[2020-01-15T02:49:37.556Z]   github.com/pingcap/dm/cmd/dm-worker.TestRunMain.func3()
[2020-01-15T02:49:37.556Z]       /home/jenkins/agent/workspace/dm_ghpr_test/go/src/github.com/pingcap/dm/cmd/dm-worker/main_test.go:64 +0x2f
[2020-01-15T02:49:37.556Z] 
[2020-01-15T02:49:37.556Z] Goroutine 72 (running) created at:
[2020-01-15T02:49:37.556Z]   github.com/pingcap/dm/dm/worker.(*Server).Start()
[2020-01-15T02:49:37.556Z]       /home/jenkins/agent/workspace/dm_ghpr_test/go/src/github.com/pingcap/dm/dm/worker/server.go:96 +0x5be
[2020-01-15T02:49:37.556Z]   github.com/pingcap/dm/cmd/dm-worker.main()
[2020-01-15T02:49:37.556Z]       /home/jenkins/agent/workspace/dm_ghpr_test/go/src/github.com/pingcap/dm/cmd/dm-worker/main.go:80 +0x6c2
[2020-01-15T02:49:37.556Z]   github.com/pingcap/dm/cmd/dm-worker.TestRunMain.func3()
[2020-01-15T02:49:37.556Z]       /home/jenkins/agent/workspace/dm_ghpr_test/go/src/github.com/pingcap/dm/cmd/dm-worker/main_test.go:64 +0x2f
[2020-01-15T02:49:37.556Z] 
[2020-01-15T02:49:37.556Z] Goroutine 56 (running) created at:
[2020-01-15T02:49:37.556Z]   github.com/pingcap/dm/cmd/dm-worker.TestRunMain()
[2020-01-15T02:49:37.556Z]       /home/jenkins/agent/workspace/dm_ghpr_test/go/src/github.com/pingcap/dm/cmd/dm-worker/main_test.go:63 +0x2bd
[2020-01-15T02:49:37.556Z]   testing.tRunner()
[2020-01-15T02:49:37.556Z]       /usr/local/go/src/testing/testing.go:909 +0x199
```

and also

```
[2020-01-15T02:49:05.014Z] --- FAIL: TestRunMain (0.00s)
[2020-01-15T02:49:05.014Z] panic: unable to redefine 's' shorthand in "break-ddl-lock" flagset: it's already used for "skip" flag [recovered]
[2020-01-15T02:49:05.014Z] 	panic: unable to redefine 's' shorthand in "break-ddl-lock" flagset: it's already used for "skip" flag
```

**Comment by WangXiangUSTC** [comment]
@kennytm I will fix it in this pr

**Pull request closed without merging**